src/main/java/com/netflix/imflibrary/st2067_2/Composition.java

```diff
@@ -1529,13 +1532,16 @@ private ByteProvider getByteProvider(ResourceByteRangeProvider resourceByteRange
                 imfErrorLogger.addError(IMFErrorLogger.IMFErrors.ErrorCodes.IMF_CPL_ERROR, IMFErrorLogger.IMFErrors.ErrorLevels.FATAL, String.format("EssenceDescriptor with Source Encoding Element %s in a track does not have a corresponding entry in the CPL's EDL", entry.getKey().toString()));
             }
             else {
-
+                referenceDOMNodeObjectModel = DOMNodeObjectModel.createDOMNodeObjectModelIgnoreSet(eDLMap.get(entry.getKey()), ignoreSet);
```

Comment:
For conformance shouldn't we have exact match including InstanceUID and EssenceLength?